### PR TITLE
Add Content Pipeline csproj

### PR DESCRIPTION
**Pull request by cra0zy**
This took me forever.

### CODE REVIEW

File: MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj [hunk 1]
```diff
@@ -0,0 +1,265 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net471</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework. The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
+
+This package provides you with the content pipeline for Windows, Mac and Linux which is used to compile raw content to xnb files.</Description>
+    <PackageTags>monogame;content;contet pipeline</PackageTags>
```
Comment (Jjagg) [inline MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj]: Typo, 'contet'.

**Comment by Jjagg** [comment]
Good call on moving 2MGFX stuff into Content.Pipeline. 👍

File: MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    <Compile Include="..\MonoGame.Framework\Utilities\Imaging\Stb.ImageWrite.Generated.cs">
+      <Link>Utilities\Imaging\Stb.ImageWrite.Generated.cs</Link>
+    </Compile>
+    <Compile Include="..\MonoGame.Framework\Content\ContentExtensions.cs">
+      <Link>Utilities\ContentExtensions.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Utilities\Imaging\" />
```
Comment (Jjagg) [inline MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj]: What's this for?

**Comment by Jjagg** [comment]
@tomspilman We need a change on the Mac build bot. Please check the Maintainer channel on Gitter.

**Comment by Jjagg** [comment]
Bump @tomspilman

On the Mac build bot, can you move the file at /Library/Frameworks/Xamarin.Mac.framework/Versions/Current/lib/mono/4.5/Facades/System.Drawing.Primitives.dll to another location? It is getting pulled in when building and causes conflicts because we use System.Drawing. Removing the facade is recommended according to [this](https://xamarin.github.io/bugzilla-archives/42/42726/bug.html) bug report.

**Comment by tomspilman** [comment]
@Jjagg - I'll get to this today...  just been buried in work this last week.

**Comment by tomspilman** [comment]
@Jjagg - That is done.

**Comment by Jjagg** [comment]
@tomspilman Could you also trigger a rebuild of this branch please? Should pass with the change :crossed_fingers:

**Comment by cra0zy** [comment]
I can do that.

**Comment by cra0zy** [comment]
@Jjagg  build is passing \o/

I need to do some more changes before we can get this merged, I'll ping you once done.

**Comment by Jjagg** [comment]
Hell yeah! Alright :)

File: MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    <Compile Include="..\PlayStation4\Tools\2MGFX\**\*.cs">
+      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </Compile>
+    <Compile Include="..\PlayStation4\Tools\MonoGame.Framework.Content.Pipeline\**\*.cs">
+      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </Compile>
+    <Compile Include="..\PSVita\Tools\2MGFX\**\*.cs">
+      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </Compile>
+    <Compile Include="..\PSVita\Tools\MonoGame.Framework.Content.Pipeline\**\*.cs">
```
Comment (Jjagg) [inline MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj]: Are you certain these paths are all good, or do we need to check with Tom?

Comment (mrhelmut) [reply MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj]: cra0zy double checked already, they're all correct. :)

**Comment by Jjagg** [comment]
Gonna merge this!

One consideration is that the size of Content.Pipeline gets seriously large with all the dependencies for the different platforms included. We should definitely **not** include binaries for every platform with the MG SDK installation. We could use the same technique as we do for DesktopGL with the RID's and possibly MSBuild properties as suggested in #6819 (if we care enough about .NET Framework, but I'd just go with RID for now).

**Pull request merged**